kratos/processes/compute_nodal_gradient_process.cpp

```diff
@@ -278,8 +289,8 @@ array_1d<double, 3>& ComputeNodalGradientProcess<ComputeNodalGradientProcessSett
     unsigned int i
     )
 {
-    array_1d<double, 3>& val = rThisGeometry[i].GetValue(mrGradientVariable);
-    return val;
+    array_1d<double, 3>& r_value = rThisGeometry[i].GetValue(*mpGradientVariable);
```

Comment:
I think you could directly return it

Reply:
Sure, but I prefer to merge now and void the travis recompilation and reapproval